test/fixtures/render/nullable.apib

```diff
@@ -0,0 +1,6 @@
+# GET /
+
++ Response 200 (application/json)
+    + Attributes
+        + s1: obihann (string, required) 
+        + s2: null (string, nullable) 
```

Comment:
Please change the Attributes in the test case to this:

```
+ s1: abc
+ s2: def (required)
+ s3 (optional)
+ s4 (nullable)
+ s5 (required, nullable)
+ s6 (optional, nullable)
```

Which should render the following JSON for body

```
{
  "s1": "abc",
  "s1": "def",
  "s4": null,
  "s5": null,
  "s6": null
}
```